Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_tools.yml

```
@@ -450,6 +480,47 @@
     autoRecharge: true
     autoRechargeRate: 30
 
+- type: entity
+  name: cyborg's UVL-21
+  id: LaserMinigunBorg
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)